data/EnumDelimiterType/_all.yml

```diff
@@ -4,6 +4,8 @@
   name: comment
 - pk: string
   name: string
+- pk: int 
```

Comment:
remove this -- this is unrelated to what you are doing.

This is for different types of delimiters only.